source/runner/DangerUtils.ts

```diff
@@ -0,0 +1,13 @@
+export function sentence(array: Array<string>): string {
+  if ((array || []).length === 0) {
+    return ""
+  }
+  if (array.length === 1) {
+    return array[0]
+  }
+  return array.slice(0, array.length - 1).join(", ") + " and " + array.pop()
+}
+
+export function href(href: string, text: string): string {
+  return `<a href="${href || "#"}">${text || ""}</a>`
```

Comment:
I'm not sure I follow with the `string | null` type. If `href` or `text` are null, would we return `null` to conform to the `string | null` return type? I figured `"#"` and `""` were sane defaults, respectively, but I was just throwing those out in case someone decides to supply `null` or `undefined` for either parameter. 🤷‍♂️